docs/refs/api/connectivity/cellular.md

```diff
@@ -4,64 +4,19 @@ The [CellularBase](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/cla
 
 Arm Mbed OS provides a reference implementation of CellularBase, which you can find [here](https://github.com/ARMmbed/mbed-os/tree/master/features/netsocket/cellular/generic_modem_driver).
 
-##### Getting started
-
-1. Choose an [Mbed board that supports cellular](https://developer.mbed.org/platforms/?mbed-enabled=15&connectivity=1), such as the [UBLOX-C027](https://developer.mbed.org/platforms/u-blox-C027/) or [MTS-DRAGONFLY](https://developer.mbed.org/platforms/MTS-Dragonfly/).
-
-1. Clone [`mbed-os-example-cellular`](https://github.com/ARMmbed/mbed-os-example-cellular). Follow the instructions in the repository.
-
-    1. Compile the code.
-    1. Flash the board.
-
-   You see output similar to the excerpt below:
-
-```
-
-mbed-os-example-cellular, Connecting...
-
-
-Connection Established.
-UDP: Sent 4 Bytes to echo.u-blox.com
-Received from echo server 4 Bytes
-
-
-Success. Exiting
-
-```
-
-##### Basic working principles
-
-You can use and extend a cellular interface and extended in various different ways. For example,
-
-- Using AT commands to control sockets in an IP stack built into the cellular modem.
-
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/Cell_AT.png)</span>
-
-- Using a PPP pipe to pass IP packets between an Mbed OS supported IP stack and cellular modem device.
-
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/Cell_PPP.png)</span>
-
-[`mbed-os-example-cellular`](https://github.com/ARMmbed/mbed-os-example-cellular) uses [a generic modem driver](https://github.com/ARMmbed/mbed-os/tree/master/features/netsocket/cellular/generic_modem_driver). Figure 2 above shows the basic design that the driver is based on. In other words, CellularInterface uses PPP. We can summarize this particular basic design as follows:
-
-* It uses an external IP stack (for example, LWIP) instead of on-chip network stacks.
-* The generic modem driver uses standard 3GPP AT 27.007 AT commands to set up the cellular modem and registers to the network.
-* After registration, the driver opens up a PPP (Point-to-Point Protocol) pipe using LWIP with the cellular modem and connects to the internet.
-
-##### CellularBase API
+##### API
 
 [![View code](https://www.mbed.com/embed/?type=library)](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/classCellularBase.html)
 
-##### Usage summary
+##### Usage
 
 To bring up the network interface:
 
 1. Instantiate an implementation of the CellularBase class (for example, the [generic modem driver](https://github.com/hasnainvirk/mbed-os/blob/cellular_feature_br/features/cellular/TARGET_GENERIC_MODEM/generic_modem_driver/)).
 1. Call the `connect(pincode, apn)` function with a PIN code for your SIM card and an APN for your network.
 1. Once connected, you can use Mbed OS [network sockets](network_sockets.md) as usual.
 
-##### Examples
-
-###### Connection establishment
+##### Example: Connection establishment
 
```

Comment:
Can you make this section a link to a repo - https://github.com/armmbed/mbed-os-example-cellular